src/app/helpers/whitelistApis.js

```diff
@@ -585,7 +585,7 @@ const API_LIST = {
     },
     '/certreg/v1/cert/reissue': {
       checksNeeded: ['ROLE_CHECK'],
-      ROLE_CHECK: [ROLE.COURSE_MENTOR]
+      ROLE_CHECK: [ROLE.CONTENT_CREATOR]
```

Comment:
Allow for both for backward compatibility
ROLE_CHECK: [ROLE.COURSE_MENTOR, ROLE.CONTENT_CREATOR]